setup.py

```diff
@@ -1,7 +1,9 @@
 """setup.py for axolotl"""
 
+import platform
 from importlib.metadata import PackageNotFoundError, version
 
+from packaging.version import Version, parse
```

Comment:
If we're going to require packaging, we probably have to update the docs so people install this before axolotl. So we'll have to update the ci to install packaging too.

Reply:
We already do say to install packaging first in the readme